src/one_dragon_qt/view/devtools/devtools_screen_manage_interface.py

```
@@ -565,3 +705,217 @@ def on_area_table_cell_clicked(self, row: int, column: int):
+            area_item = self.chosen_screen.area_list[row_index]
+            area_item.pc_rect = old_rect
+
+            # 更新表格显示
+            self.area_table.blockSignals(True)
+            self.area_table.item(row_index, 2).setText(f'({old_rect.x1}, {old_rect.y1}, {old_rect.x2}, {old_rect.y2})')
+            self.area_table.blockSignals(False)
+
+            # 更新图像显示
+            self._image_update.signal.emit()
+
+    def _apply_redo(self, change_record: dict[str, Any]) -> None:
```

Ruff (0.12.2)

Unused method argument: `change_record`

(ARG002)